docs/development/release.rst

```diff
@@ -90,9 +90,8 @@ Steps for the day to announce the release:
    (decide on a case by case basis, if it's relevant to the group of people):
 
     * https://groups.google.com/forum/#!forum/astropy-dev
-    * https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro
-    * CTA DATA list (cta-wp-dm@cta-observatory.org)
-    * hess-analysis
+    * CTAO AS WG list (cta-wg-as@cta-observatory.org)
+    * hess-forum list (hess-forum@lsw.uni-heidelberg.de)
```

Comment:
Should be list plain internal email addresses in a docs webpage?

Reply:
Very good point